lib/livebook_web/live/hub/teams/deployment_group_list_component.ex

```diff
@@ -29,7 +29,7 @@ defmodule LivebookWeb.Hub.Teams.DeploymentGroupListComponent do
               <.menu_item>
                 <.link
                   id={"hub-deployment-group-#{deployment_group.id}-edit"}
-                  patch={~p"/hub/#{@hub_id}/deployment-groups/edit/#{deployment_group.id}"}
+                  navigate={~p"/hub/#{@hub_id}/deployment-groups/edit/#{deployment_group.id}"}
```

Comment:
Why did we change this to navigate? There is another PR changing everything to patch. We should try to be consistent here.

Reply:
Done!